### PR TITLE
secondary-section for mobile

### PR DESCRIPTION
**Pull request by kuldeep3**
#29 
image

1. Added secondary section for mobile view.
2. Secondary section includes card view, with button label and URL which can be added from the dashboard.
3. Modified position of the primary container of the page.

### CODE REVIEW

File: public/wp-content/themes/glific/page-get-started.php [hunk 1]
```diff
@@ -52,13 +52,13 @@
...
+                <div class="d-flex flex-column w-xl-426 w-md-360 p-8 mb-10 mb-md-0 p-xl-10 p-md-9 bg-white get-started-content-box rounded-top-30 rounded-bottom-right-30">
+                    <h2 class="fz-18 fz-md-24 w-xl-345 w-md-310"><?php echo $data['title']; ?></h2>
                     <?php if (!empty($data['button_url']) && !empty($data['button_text'])) : ?>
-                        <div class="d-flex flex-row justify-content-center mt-xl-7 mt-md-8">
-                            <a href="<?php $data['button_url']; ?>" class="d-block w-232 text-decoration-none bg-theme-secondary text-white font-heebo-regular fz-18 px-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15"><?php echo $data['button_text']; ?></a>
+                        <div class="d-flex flex-row justify-content-center mt-5 mt-xl-7 mt-md-8">
+                            <a href="<?php $data['button_url']; ?>" class="d-block w-full w-md-232 text-decoration-none bg-theme-secondary text-white font-heebo-regular fz-18 px-8 px-md-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15"><?php echo $data['button_text']; ?></a>
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/page-get-started.php]: @kuldeep3 try breaking down these larger lines into small lines.
```suggestion
                            <a href="<?php $data['button_url']; ?>"
                                class="d-block w-full w-md-232 text-decoration-none bg-theme-secondary text-white font-heebo-regular fz-18 px-8 px-md-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15">
                                    <?php echo $data['button_text']; ?>
                            </a>
```

Comment (Satendra-SR) [reply public/wp-content/themes/glific/page-get-started.php]: Check the other places as well. Also, can you try finding some plugin that can automatically do it for PHP. Please share with the team.

File: public/wp-content/themes/glific/src/scss/style.scss [hunk 2]
```diff
@@ -87,5 +87,6 @@ body{
 @import 'media/xl';
 @import 'media/hd';
 @import 'header/style';
-@import 'page/get-started.scss';
-@import 'page/faq.scss';
+@import 'page-get-started/style';
+@import 'page-faq/style';
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/style.scss:91]: 👍

File: public/wp-content/themes/glific/src/scss/utilities/xl.scss [hunk 3]
```diff
@@ -55,4 +55,30 @@
 			border-bottom-left-radius: $value;
 		}
 	}
+	@each $name, $value in $spacers{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/utilities/xl.scss:58]: 👍

**Review by Satendra-SR — APPROVED** [review]
@kuldeep3 just one issue, else looks good. 
If you found something on formating, could you please add it to [ColoredCow Resources](https://github.com/ColoredCow/resources/tree/master/php) ?

File: public/wp-content/themes/glific/page-get-started.php (hunk 1 above)
Comment (kuldeep3) [reply public/wp-content/themes/glific/page-get-started.php]: @Satendra-SR Sure, let me search a little bit, if found I will let you know and the team.

**Pull request merged**